python/cuml/experimental/explainer/__init__.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.experimental.explainer.permutation_shap import PermutationSHAP
```

Comment:
Should we rename this to like PermutationExplainer or something similar to match the SHAP explainer naming, just as we match skl names?

Reply:
One of those things that I had in mind for a while. I like that it says SHAP so its very clear what explainer it is, but also parity with the package would be good, so I'm very neutral regarding this. Probably parity with the main package would be better?